docker/Dockerfile

```diff
@@ -1,30 +1,14 @@
-FROM osgeo/gdal:ubuntu-full-v2.5.0RC1
-ARG PYTHON_VERSION=3.6
+FROM python:3
 
-RUN apt-get update && apt-get install -y software-properties-common
```

Comment:
I couldn't tell how much of the stuff that was here was is use, so I started small and layered in the `RUN` instructions I needed to get the notebooks working. I may have cause some regressions with `usa_hcris2018_facilitybedcounts_20200313_v1.ipynb`, but unsure.